layers/+tools/google-calendar/README.org

```diff
@@ -0,0 +1,166 @@
+#+TITLE: Google calendar layer
```

Comment:
Would it make sense to have this be the general calendar layer and have org-gcal support just be optional (if a var is set, for example)